Makefile

```diff
@@ -32,15 +32,13 @@ depend: work
 ifndef HAS_MERCURIAL
 	pip install Mercurial
 endif
-ifndef HAS_GLIDE
-	go get -u github.com/Masterminds/glide
-endif
-ifeq ($(wildcard $(DEST)/vendor/.*),)
-		cd $(DEST) && glide install --strip-vendor
+ifndef HAS_DEP
+	go get -u github.com/golang/dep/cmd/dep
```

Comment:
I'm not sure we should use the dev branch of dep here as opposed to pulling/installing an official release.  I'll let others comment on that, but the dep page does state pretty clearly that you shouldn't use the master branch for production etc.

Reply:
That's true... I think I will go your way and curl the install.sh script. That should be platform independent and use a stable release.